mne/forward/_lead_dots.py

```diff
@@ -253,9 +297,69 @@ def _do_self_dots_subset(intrad, rmags, rlens, cosmags, ws, volume, lut,
     return products
 
 
+def _do_cross_dots(intrad, volume, coils1, coils2, r0, ch_type,
+                   lut, n_fact):
```

Comment:
Please add the file where you found this -- I might have forgetten to do that with some of these so I recognize this is a hypocritical critique :) If you feel like adding the ones that I forgot, feel free, but you don't have to. It's just nice for the next person to have the trail of breadcrumbs to follow.